test/views/after_auth_screens/events/create_event_page_test.dart

```diff
@@ -354,52 +354,52 @@ void main() {
     expect((tester.widgetList(switches).toList()[1] as Switch).value, false);
   });
 
-  testWidgets("Checking tap Inkwell work for admin list", (tester) async {
-    await tester.pumpWidget(
-      createEventScreen(
-        themeMode: ThemeMode.dark,
-        theme: TalawaTheme.darkTheme,
-      ),
-    );
-    await tester.pump();
-    final inkwellFinder = find.byType(InkWell);
-    expect(inkwellFinder, findsNWidgets(8));
-
-    ///returning the file variable to the
-    ///result of function multimediaPickerService.getPhotoFromGallery
-    ///when this function is called in the
-    ///view model of add_post_page.
-    final file = File('fakePath');
-
-    /// using the new instance of multimediaPickerService
-    /// so that when statement can be used again,
-    /// else it gives null point exception
-    final multimediaPickerService = locator<MultiMediaPickerService>();
-
-    /// when is function provided by mockito lib
-    when(multimediaPickerService.getPhotoFromGallery(camera: false))
-        .thenAnswer((_) async {
-      return file;
-    });
-
-    await tester.ensureVisible(find.byKey(const Key('inwell_cep1')));
-    await tester.pumpAndSettle();
-
-    await tester.tap(find.byKey(const Key('inwell_cep1')));
-    await tester.pump();
-
-    //TODO: implement Rest of the test when the _adminCheckedMap or _memberCheckedMap return true for some id
-    //TODO: for not it return false for all the ordMember.id
-    //
-    // orgMembersList.forEach((orgMember) {
-    //   if (isAdmin) {
-    //     _adminCheckedMap.putIfAbsent(orgMember.id!, () => false);
-    //   } else {
-    //     _memberCheckedMap.putIfAbsent(orgMember.id!, () => false);
-    //   }
-    //   _memberCheckedMap.putIfAbsent(orgMember.id!, () => false);
-    // });
-  });
+  // testWidgets("Checking tap Inkwell work for admin list", (tester) async {
```

Comment:
This is not appreciated @Kushalrock please don't comment out tests please try fixing them.

Reply:
@CyberWake That is the test for admins part of the screen which has been removed.